docs/examples/overture-tiles.mdx

```diff
@@ -78,6 +78,14 @@ npm run cdk deploy
 
   * Select the **OvertureTilesQueue** as the Job queue.
 
+  * Input the **Command** as a JSON list: `["2024-07-22.0","my-bucket","addresses"]`
+
+    * The first item is the release version, including minor version: `2024-07-22.0`
+
+    * The second item is the target bucket: `my-bucket`
+
+    * The third item is the theme: `addresses`
+
   * Once the job is complete, the tileset will be available at `s3://BUCKET_NAME/RELEASE/THEME.pmtiles`.
```

Comment:
Consider elaborating that `$RELEASE` is trimmed (e.g. `2024-07-22`) when published to S3 to enable patching of releases.

Reply:
updated, along with 2024-08-20 URLs!

Reply:
do we have any better nomenclature for `2024-08-20` vs `2024-08-20.0` ? I called the later the "Data release with minor version", and the former just `RELEASE`